src/scripts/main.js

```diff
@@ -1 +1,104 @@
 'use strict';
+
+function promise1() {
+  const resolver = (confirm1, reject1) => {
+    document.addEventListener('contextmenu', () => {
+      return confirm1('First promise was resolved');
+    });
+
+    setTimeout(() => {
+      return reject1('First promise was rejected');
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+promise1()
+  .then(result => {
+    const div = document.querySelector('body');
+
+    div.innerHTML
+    += `<div class="success"
+     data-qa="notification">
+     ${result}</div>`;
```

Comment:
create a helper function, which creates a message and append it to body

Reply:
so that it looks something like this

```
firstPromise
  .then((message) => {
    createNotification(message, 'success');
  })
  .catch((message) => {
    createNotification(message, 'error');
  });
```